### PR TITLE
chore: update pages with third party faucets and removed zksync portal and faucets mentions

### PR DESCRIPTION
**Pull request by amelnytskyi**
# What :computer: 
* Replaced all the goerli.portal.zksync.io/faucet and portal.zksync.io/ mentions
* Updated third party faucets list

# Why :hand:
* Reason why first thing was added to PR
* Reason why second thing was added to PR
* Reason why third thing was added to PR

# Evidence :camera:
Include screenshots, screen recordings, or `console` output here demonstrating that your changes work as intended

# Notes :memo:
* Any notes/thoughts that the reviewers should know prior to reviewing the code?

### CODE REVIEW

File: docs/dev/building-on-zksync/hello-world.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ This is what we're going to do:
   - [Alchemy Goerli faucet](https://goerlifaucet.com/)
   - [Paradigm Goerli faucet](https://faucet.paradigm.xyz/)
   - [Proof of work faucet](https://goerli-faucet.pk910.de/)
-- ERC20 tokens on zkSync are required for the testnet paymaster. We recommend using [the faucet from the zkSync portal](https://goerli.portal.zksync.io/faucet).
+- ERC20 tokens on zkSync are required for the testnet paymaster. Get testnet `ETH` on zkSync Era using [bridges](https://zksync.io/explore#bridges) to bridge funds to zkSync.
```
Comment (githubdoramon) [inline docs/dev/building-on-zksync/hello-world.md:30]: "on"-> "for"

Comment (githubdoramon) [reply docs/dev/building-on-zksync/hello-world.md:30]: Ah, and given we are talking about testnet tokens, people will need to swap them on some dex. Worth pointing them to some, or point them directly to the token contracts (assuming they can mint directly)

File: docs/dev/tutorials/aa-daily-spend-limit.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ The daily limit feature prevents an account from spending more ETH than the limi
     - [Alchemy Goerli faucet](https://goerlifaucet.com/)
     - [Paradigm Goerli faucet](https://faucet.paradigm.xyz/)
     - [Proof of work faucet](https://goerli-faucet.pk910.de/)
-  - Get testnet `ETH` on zkSync Era from the [zkSync portal](https://goerli.portal.zksync.io/faucet).
+  - Get testnet `ETH` on zkSync Era using [bridges](https://zksync.io/explore#bridges) to bridge funds to zkSync.
```
Comment (githubdoramon) [inline docs/dev/tutorials/aa-daily-spend-limit.md:25]: on -> for

File: docs/dev/tutorials/api3-usd-paymaster-tutorial.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ This tutorial shows you how to build a custom paymaster that allows users to pay
     - [Alchemy Goerli faucet](https://goerlifaucet.com/)
     - [Paradigm Goerli faucet](https://faucet.paradigm.xyz/)
     - [Proof of work faucet](https://goerli-faucet.pk910.de/)
-  - Get testnet `ETH` on zkSync Era from the [zkSync portal](https://goerli.portal.zksync.io/faucet).
+  - Get testnet `ETH` on zkSync Era using [bridges](https://zksync.io/explore#bridges) to bridge funds to zkSync.
```
Comment (githubdoramon) [inline docs/dev/tutorials/api3-usd-paymaster-tutorial.md:29]: on -> for

File: docs/dev/tutorials/cross-chain-tutorial.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ This tutorial shows you how to implement communication between L1 and L2 with th
     - [Alchemy Goerli faucet](https://goerlifaucet.com/)
     - [Paradigm Goerli faucet](https://faucet.paradigm.xyz/)
     - [Proof of work faucet](https://goerli-faucet.pk910.de/)
-  - Get testnet `ETH` on zkSync Era from the [zkSync portal](https://goerli.portal.zksync.io/faucet).
+  - Get testnet `ETH` on zkSync Era using [bridges](https://zksync.io/explore#bridges) to bridge funds to zkSync.
```
Comment (githubdoramon) [inline docs/dev/tutorials/cross-chain-tutorial.md:26]: on -> for

File: docs/tools/hardhat/getting-started.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ If you are using Vyper, check out the [Vyper plugin documentation](./hardhat-zks
     - [Alchemy Goerli faucet](https://goerlifaucet.com/)
     - [Paradigm Goerli faucet](https://faucet.paradigm.xyz/)
     - [Proof of work faucet](https://goerli-faucet.pk910.de/)
-  - Get testnet `ETH` on zkSync Era from the [zkSync portal](https://goerli.portal.zksync.io/faucet).
+  - Get testnet `ETH` on zkSync Era using [bridges](https://zksync.io/explore#bridges) to bridge funds to zkSync.
```
Comment (githubdoramon) [inline docs/tools/hardhat/getting-started.md:43]: on -> for

File: docs/tools/hardhat/hardhat-zksync-deploy.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ To use the `hardhat-zksync-deploy` in your project, we recommend that:
 
 - You have a Node installation and `yarn` package manager.
 - You are already familiar with deploying smart contracts on zkSync Era. If not, please refer to the first section of the [quickstart tutorial](../../dev/building-on-zksync/hello-world.md).
-- You have a wallet with sufficient Göerli `ETH` on L1 to pay for bridging funds to zkSync as well as deploying smart contracts. We recommend using [our faucet from the zkSync portal](https://goerli.portal.zksync.io/faucet).
+- You have a wallet with sufficient Göerli `ETH` on L1 to pay for bridging funds to zkSync as well as deploying smart contracts. Use the [third party faucets](../../reference/troubleshooting/faq.md#how-do-i-request-funds-for-testnet) to get some test ERC20 tokens in your account.
```
Comment (githubdoramon) [inline docs/tools/hardhat/hardhat-zksync-deploy.md:18]: shouldn't mention ERO 20s on this sentence

File: docs/tools/hardhat/hardhat-zksync-vyper.md [hunk 7]
```diff
@@ -22,7 +22,7 @@ Learn more about the latest updates in the [changelog](https://github.com/matter
     - [Alchemy Goerli faucet](https://goerlifaucet.com/)
     - [Paradigm Goerli faucet](https://faucet.paradigm.xyz/)
     - [Proof of work faucet](https://goerli-faucet.pk910.de/)
-  - Get testnet `ETH` on zkSync Era from the [zkSync portal](https://goerli.portal.zksync.io/faucet).
+  - Get testnet `ETH` on zkSync Era using [bridges](https://zksync.io/explore#bridges) to bridge funds to zkSync.
```
Comment (githubdoramon) [inline docs/tools/hardhat/hardhat-zksync-vyper.md:25]: on -> for

**Review by githubdoramon — CHANGES_REQUESTED** [review]

File: docs/dev/building-on-zksync/hello-world.md (hunk 1 above)
Comment (amelnytskyi) [reply docs/dev/building-on-zksync/hello-world.md]: Fixed

File: docs/dev/tutorials/aa-daily-spend-limit.md (hunk 2 above)
Comment (amelnytskyi) [reply docs/dev/tutorials/aa-daily-spend-limit.md]: Fixed

File: docs/dev/tutorials/api3-usd-paymaster-tutorial.md (hunk 3 above)
Comment (amelnytskyi) [reply docs/dev/tutorials/api3-usd-paymaster-tutorial.md]: Fixed

File: docs/tools/hardhat/getting-started.md (hunk 5 above)
Comment (amelnytskyi) [reply docs/tools/hardhat/getting-started.md]: Fixed

File: docs/tools/hardhat/hardhat-zksync-deploy.md (hunk 6 above)
Comment (amelnytskyi) [reply docs/tools/hardhat/hardhat-zksync-deploy.md]: Fixed

File: docs/tools/hardhat/hardhat-zksync-vyper.md (hunk 7 above)
Comment (amelnytskyi) [reply docs/tools/hardhat/hardhat-zksync-vyper.md]: Fixed

**Comment by amelnytskyi** [comment]
@githubdoramon @idea404 requested changes added - could you please review again?

**Review by githubdoramon — APPROVED** [review]

**Review by idea404 — APPROVED** [review]

**Pull request merged**